src/layouts/ThinLayout/ThinLayout.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file ThinLayout.js
+ * Exports a React component that renders EditVR's thin layout.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core';
+import LoadingBar from 'react-redux-loading-bar';
+
+import { ThemeProvider } from '../../hoc';
+import { Header, Footer } from '../';
+import ThinLayoutStyles from './ThinLayout.style';
+
+const ThinLayout = ({ children, classes }) => (
+  <div id="layout__wrapper" className={classes.wrapper}>
+    <LoadingBar style={{ backgroundColor: '#FFFFFF' }} />
```

Comment:
I know adding inline styles isn't great, but:

Reply:
super cool. I love it when pieces come together like this!